<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color:#292C34;
color:white;
}</string>
  </property>
  <widget class="QComboBox" name="dropDown">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
border: 1px solid white;
border-radius: 5px;
background-color: #292C34;
color: white;
padding: 5px;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>89</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 1px solid white;
border-radius: 5px;
background-color: #292C34;
color: white;
}</string>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="loopCheckBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>64</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Loop?</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="spacebarCheckBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Change playback on Spacebar,
 go back on Backspace,
 and skip using Enter</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
